### PR TITLE
[SDK-266] feat: update readme with docs from gitbooks

### PR DESCRIPTION
**Pull request by HarrisonHough**
## Description

-  As part of updating the Gitbooks docs for WebView I thought it would also be useful to have the same docs here in the README

#### Updated

-   documentation to README.md

## Checklist

-   [ ] Tests written or updated for the changes.
-   [ ] Documentation is updated.
-   [ ] Changelog is updated.

[SDK-266]: https://ready-player-me.atlassian.net/browse/SDK-266?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,3 +16,71 @@ Please visit the online documentation and join our public `discord` community.
...
+
+Creating a Scene with a WebView in your Unity project is the same for Android and iOS.
+
+1. Create or open your Unity project.
+2. Import the Ready Player Me Core package into your project, if you haven't done so already.
+3. Navigate to the Unity package manager by going to Window > Package Manager.
+4. Find the Ready Player Me WebView package under Packages - Ready Player Me.
+5. Find and import the WebView Sample by clicking the Import button.
+6. Now open the WebVew example scene by finding it in the Projects tab under `Samples/Ready Player Me WebView/\[VERSION_NUMBER\]/WebView`.
+7. In the Hierarchy tab, you will find a WebView Canvas object withe a child object called WebView Panel, select the WebVewPanel.
```
Comment (rYuuk) [inline README.md:40]: typo in withe

Comment (rYuuk) [inline README.md:40]: WebVew typo

**Review by rYuuk — APPROVED** [review]
🟢 Please look at some typos.

File: README.md (hunk 1 above)
Comment (HarrisonHough) [reply README.md]: fixed

Comment (HarrisonHough) [reply README.md]: fixed

**Pull request merged**